.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: c
```

Comment:
I don't know how travis works. Is this `language: c` on purpose?

Reply:
Yes, unfortunately Travis has no support for OCaml, so what happens is we start with the C setup and install compilers etc. using the [ocaml-ci-scripts](https://github.com/ocaml/ocaml-ci-scripts) tooling which handles all the tedious stuff like installing dependencies from `opam` files, running tests etc.